### PR TITLE
Docs/update setup run and test steps

### PR DESCRIPTION
**Pull request by benbierens**

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -121,6 +147,25 @@ $ make test
...
+Before the integration tests are started, you must start the Ethereum test node manually.
+1. Open a CMD terminal
+1. Go to the vendor/codex-contracts-eth folder: `cd /<git-root>/vendor/codex-contracts-eth/`
+1. `npm start` -> Should output a number of keys and a warning message.
+
+#### Run
+
 The `testAll` target runs the same tests as `make test` and also runs tests for nim-codex's Ethereum contracts, as well a basic suite of integration tests.
 
 To run `make testAll`, Node.js needs to be installed. [Node Version Manager (`nvm`)](https://github.com/nvm-sh/nvm#readme) is a flexible means to do that and it works on Linux, macOS, and Windows + MSYS2.
```
Comment (markspanbroek) [inline BUILDING.md:171]: Can you integrate these instructions for macOS and Linux (and Windows?) with the new instructions that you wrote?

File: TWOCLIENTTEST.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Steps
+
+### 1. Launch Node #1
+
+Open a terminal and run:
+- Mac/Unx: `"build/codex" --data-dir="$(pwd)\Data1" --listen-addrs=/ip4/127.0.0.1/tcp/8070 --api-port=8080  --disc-port=8090`
+- Windows: `"build/codex.exe" --data-dir="Data1" --listen-addrs=/ip4/127.0.0.1/tcp/8070 --api-port=8080 --disc-port=8090`
+
+(Hint: If your terminal interprets the '/' in the listen-address as a reference to your root path, try running the command from a shell-script!)
```
Comment (markspanbroek) [inline TWOCLIENTTEST.md:17]: Can you add quotes in Windows to avoid this?

**Review by markspanbroek — APPROVED** [review]
Nice, thanks for updating these instructions!

File: TWOCLIENTTEST.md (hunk 2 above)
Comment (benbierens) [reply TWOCLIENTTEST.md:17]: Tried that, it doesn't like it. I added a hint for the one way I found to get this to work.

**Comment by benbierens** [comment]
> I would actually suggest we merge the build and test docs into the readme - it's obviously a preference thing, but I think a large doc gives you better at a glance experience and is easier to search? Just a suggestion.

I propose to keep the files separate, and use MD links to direct readers to where they want to go. I prefer many 'small' but specific files over one document that describes everything.

**Review by markspanbroek — APPROVED** [review]
Nice!

**Pull request merged**